arch/portduino/portduino.ini

```diff
@@ -34,4 +34,9 @@ build_flags =
   -DPORTDUINO_LINUX_HARDWARE
   -lbluetooth
   -lgpiod
-  -lyaml-cpp
\ No newline at end of file
+  -lyaml-cpp
+  -lssl
+  -lorcania
```

Comment:
The problem with this approach is that we would like to keep this buildable for distros that don't package orcania/ulfius. See https://github.com/meshtastic/firmware/blob/master/variants/portduino/platformio.ini#L6C4-L6C31

I'm surprised the inclusion of those lines in the variants/portduino/platformio.ini isn't sufficient.

Reply:
This has been fixed in the PR now and those lines removed from portuino.ini
Confirmed my builds work successfully with webserver now.